relayer-cli/src/utils/relayerHelpers.ts

```diff
@@ -0,0 +1,51 @@
+import * as fs from "fs";
+
+async function initialize(chain_id: number, network: string): Promise<number> {
+  const lock_file_name = "./src/state/" + network + "_" + chain_id + ".pid";
```

Comment:
Some suggestions on this line:
- `src` should contain only code, not data. Consider using a directory like `data` or `state` that only contains this kind of autogenerated files. It will also make it way easier to dockerize the repo. Ideally the base directory path for data would be provided via env var.
- Please use `camelCase` for variables as it is more idiomatic in JS

Reply:
yes! feel free to open an issue for that, as I assume it is not relevant to this PR only

Reply:
Fixed in this PR